swagger.json

```diff
@@ -568,25 +568,24 @@
                 }
             }
         },
-        "/fleet/drivers/{driver_id}": {
-            "parameters": [
+        "/fleet/drivers/{driver_id OR external_id}": {
+            "get": {
+              "parameters": [
                 {
                     "$ref": "#/parameters/accessTokenParam"
                 },
                 {
-                    "name": "driver_id",
+                    "name": "driver_id OR external_id",
```

Comment:
Could we move this to the common `#/parameters/` section, so this can be re-used across HTTP methods?